plugins/ca/qml/Keyboard_ca_email.qml

```diff
@@ -34,30 +34,30 @@ KeyPad {
 
             CharKey { label: "q"; shifted: "Q"; extended: ["1"]; extendedShifted: ["1"]; leftSide: true; }
             CharKey { label: "w"; shifted: "W"; extended: ["2"]; extendedShifted: ["2"] }
-            CharKey { label: "e"; shifted: "E"; extended: ["3", "è", "é", "ê", "ë", "€"]; extendedShifted: ["3", "È","É", "Ê", "Ë", "€"] }
```

Comment:
I would not remove these characters from the email keyboard. It is possible to have an email address with these characters in them, even if it is not very common. It's better to have the option for the (incredibly unlikely) case than to not have it, in this case. This possibility was added in [RFC 6532](https://tools.ietf.org/html/rfc6532).

Instead, match these accents to those contained in the standard keyboard.

Reply:
Roger. I'll fix that

Reply:
I have added the characters from regular layout except the pronouns